modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -657,6 +674,9 @@ protected String getHydratedModuleName() {
 		props.put("selectAllURL", getSelectAllURL());
 		props.put("selectable", isSelectable());
 		props.put("showCreationMenu", isShowCreationMenu());
+		props.put(
+			"showDesignImprovements",
```

Comment:
It would may be a good idea to rename this to `showDesignImprovementsFF`? What do you think?

Reply:
Sure, makes it clearer that this is for a FF. On it!